domenetjenester/fpsak/src/main/java/no/nav/foreldrepenger/fpsak/dto/kodeverk/Kode.java

```diff
@@ -0,0 +1,28 @@
+package no.nav.foreldrepenger.fpsak.dto.kodeverk;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Kode {
+    public String kodeverk;
+    public String kode;
+    public String navn;
+
+    public Kode() {
+    }
+
+    public Kode(String kodeverk, String kode, String navn) {
```

Comment:
Finnes det / kan ikke vi ha en mer generell DTO for kodeverk? Det er vel ikke kun brukt for fpsak?